static/app/gettingStartedDocs/python/django.tsx

```diff
@@ -2,13 +2,36 @@ import ExternalLink from 'sentry/components/links/externalLink';
 import {Layout, LayoutProps} from 'sentry/components/onboarding/gettingStartedDoc/layout';
 import {ModuleProps} from 'sentry/components/onboarding/gettingStartedDoc/sdkDocumentation';
 import {StepType} from 'sentry/components/onboarding/gettingStartedDoc/step';
+import {ProductSolution} from 'sentry/components/onboarding/productSelection';
 import {t, tct} from 'sentry/locale';
 
 // Configuration Start
+
+const profilingConfiguration = `
+# To set a uniform sample rate
+# Set profiles_sample_rate to 1.0 to profile 100%
+# of sampled transactions.
+# We recommend adjusting this value in production,
+profiles_sample_rate=1.0,
+`;
```

Comment:
In the full documentation version, there's an optional setup step for profiling. You can find it [here](https://docs.sentry.io/platforms/python/profiling/?#enable-profiling). Should we consider adding this step as well? How useful is it for new users?

```
   # Alternatively, to control sampling dynamically
    profiles_sampler=profiles_sampler
```

Reply:
cc @ale-cota

Reply:
We can leave this out. That's true also for Performance, but it's already an "advanced setting", so no need to put it in onboarding.